resume-buddy/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -33,14 +35,18 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
+  /* TODO: find a way for the user type to be part of the HttpServletRequest */
```

Comment:
Change this TODO to:

`Find a way for the user type to be passed to the CommentsServlet from JavaScript (instead of being stored here as a static variable). Maybe try using the localStorage web API: https://stackoverflow.com/questions/12895754/how-to-save-the-state-of-a-javascript-variable-that-is-used-in-multiple-html-pag